src/components/Avatar/index.ts

```diff
@@ -0,0 +1 @@
+export * from './avatar.component'
```

Comment:
Essa é uma outra prática que confesso não sou particularmente fã, além de você estar criando um arquivo a mais que no final não faz basicamente nada, se tiver qualquer coisinha dentro do componente que não for utilizada, pelo fato de usar `*` poderá acabar sendo carregado para o bundle se o three shaking num tiver bem configurado.